### DISCUSSION TITLE
Marketplace Ratings and Review UI enhancement

### DISCUSSION THREAD

**Discussion by amit-kulkarni-in** [microsoft/vscode-discussions#329] [Announcements]
We have made minor enhancements to the Ratings and Review page in the extension by adding "Report Issue" button. This would allow users to divert to either github issues link or support link or email address who want to communicate with the extension authors on the issues instead of providing lower ratings. This action will only be rendered for extensions that have specified a `bugs` property in their `package.json`.

![Report Issue](https://user-images.githubusercontent.com/104777722/208937923-1e3f4982-a229-4dff-af34-bc8ab3308b48.png)

We hope this would allow users to report issue at the right place instead of leaving bug comments on Ratings and Review.  In our data we are already seeing around 10%+ of users choosing the Report Issue button.

**Comment by andyleejordan** [comment]
Do you think it's better for that `bugs` property to point to the [list of all issues](https://github.com/PowerShell/vscode-powershell/issues), or the [create new issue flow](https://github.com/PowerShell/vscode-powershell/issues/new/choose)?
